packages/mobile/src/assets/images/iconDownloadGray.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="32" height="33" viewBox="0 0 32 33" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
fill maybe should be #000 to make dark mode color setting work? (not merge blocking)